presto-main/src/main/java/com/facebook/presto/server/QuerySessionSupplier.java

```diff
@@ -64,13 +65,23 @@ public QuerySessionSupplier(
     }
 
     @Override
-    public Session createSession(QueryId queryId, SessionContext context, WarningCollectorFactory warningCollectorFactory)
+    public Session createSession(QueryId queryId, SessionContext context, WarningCollectorFactory warningCollectorFactory, Optional<AuthorizedIdentity> authorizedIdentity)
     {
         Identity identity = context.getIdentity();
+        if (authorizedIdentity.isPresent()) {
```

Comment:
Do you think we should log that the identity was updated? Or this information would be available in other places as well?

Reply:
I used logging for test and removed it. Maybe we should log before we log it to the scuba. Let me do it.

Reply:
Done.